.github/workflows/CI.yml

```diff
@@ -43,30 +60,49 @@ jobs:
   make_mason:
     runs-on: ubuntu-latest
     env:
+      # To try to keep this config from running the longest,
+      # use a simpler configuration
+      CHPL_TASKS: fifo
+      CHPL_MEM: cstdlib
+      CHPL_GMP: none
+      CHPL_RE2: bundled
+      CHPL_LLVM: none
+      # check that mason builds
       CHPL_SMOKE_SKIP_MAKE_MASON: false
+      # run make check in this config too
+      # to get more coverage for quickstart and because it's
+      # not the slowest config
+      CHPL_SMOKE_SKIP_MAKE_CHECK: false
```

Comment:
Not a problem you introduced and not something to fix in this PR, but it would be a lot cleaner if instead of env vars we could just do `./util/buildRelease/smokeTest check mason` or something.

Reply:
Yeah. I was half a mind to make this CI stuff just run e.g. `make mason` but the smokeTest script does take some other actions.